tests/Behat/Gherkin/Filter/LineFilterTest.php

```diff
@@ -71,19 +71,20 @@ public function testFilterFeatureScenario()
         $this->assertCount(0, $scenarios = $feature->getScenarios());
     }
 
-    public function testFilterFeatureOutline()
+    public function testFilterFeatureOutline(): void
     {
         $filter = new LineFilter(13);
         $feature = $filter->filterFeature($this->getParsedFeature());
-        /* @var OutlineNode[] $scenarios */
         $this->assertCount(1, $scenarios = $feature->getScenarios());
         $this->assertSame('Scenario#3', $scenarios[0]->getTitle());
+        $this->assertInstanceOf(OutlineNode::class, $scenarios[0]);
```

Comment:
These `assertInstanceOf()` are a convenient way to (phpunit-)test the code as well as tell phpstan that the `ScenarioInterface` object is in fact an `OutlineNode`.